components/raftstore-v2/src/operation/command/write/mod.rs

```diff
@@ -295,19 +293,23 @@ impl<EK: KvEngine, R: ApplyResReporter> Apply<EK, R> {
                 );
             }
 
-            let _ = wait.await;
-        }
+            wait.await.unwrap()
+        } else {
+            false
+        };
 
         info!(
             self.logger,
             "execute delete range";
             "range_start" => log_wrappers::Value::key(&start_key),
             "range_end" => log_wrappers::Value::key(&end_key),
             "notify_only" => notify_only,
-            "use_delete_range" => use_delete_range,
             "duration" => ?start.saturating_elapsed(),
         );
 
+        if index != u64::MAX && written {
```

Comment:
When is it `u64::MAX`?

Reply:
I don't know, it's the old code, shared by all write requests.